packages/manager/apps/web-ongoing-operations/src/components/Dashboard/DashboardPage.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import '@/setupTests';
+import { vi } from 'vitest';
+import { render, waitFor } from '@testing-library/react';
+import { useResourcesIcebergV6 } from '@ovh-ux/manager-react-components';
+import React from 'react';
+import { wrapper } from '@/utils/test.provider';
+import { useGetDomainInformation } from '@/hooks/data/query';
+import { serviceInfo } from '@/__mocks__/serviceInfo';
+import DashboardPage from '@/components/Dashboard/DashboardPage';
+import { ParentEnum } from '@/enum/parent.enum';
+import { taskMeDomain } from '@/constants';
+import { domain } from '@/__mocks__/domain';
+
+vi.mock('react-router-dom', () => ({
+  useNavigate: () => vi.fn(() => null),
+  Navigate: vi.fn(() => null),
+  useLocation: vi.fn(),
+}));
+
+vi.mock('@/hooks/data/query', () => ({
+  useGetDomainInformation: vi.fn(),
+}));
+
+describe('Datagrid template', () => {
+  it('displays loading spinner while main requests are loading', async () => {
```

Comment:
```suggestion
  it('displays loading spinner while main requests are loading', () => {
```
no needed if you don't have "await"